packages/scene-composer/src/components/panels/AddComponentMenu.tsx

```diff
@@ -31,14 +32,15 @@ type AddComponentMenuItem = ToolbarItemOptions & {
 const labelStrings: { [key in ObjectTypes]: MessageDescriptor } = defineMessages({
   [ObjectTypes.Component]: { defaultMessage: 'Add component', description: 'Menu Item label' },
   [ObjectTypes.Overlay]: { defaultMessage: 'Overlay', description: 'Menu Item label' },
-  [ObjectTypes.EntityBinding]: { defaultMessage: 'Add entity binding', description: 'Menu Item label' },
+  [ObjectTypes.DataBinding]: { defaultMessage: 'Add entity binding', description: 'Menu Item label' },
```

Comment:
What's the reason for changing the. exisitng logic? 
```[ObjectTypes.EntityBinding] to [ObjectTypes.DataBinding]```

Reply:
o I think that was caused by me syncing the branch that is not supposed to be there